py-polars/polars/io/database/_executor.py

```diff
@@ -172,13 +171,17 @@ def _from_arrow(
         infer_schema_length: int | None,
     ) -> DataFrame | Iterable[DataFrame] | None:
         """Return resultset data in Arrow format for frame init."""
+        from polars import DataFrame
```

Comment:
I'd prefer to have imports at the top of the file if possible. We can use the `import polars._reexport as pl` and use `pl.DataFrame`, right? Or does that also lead to circular imports?

Reply:
Unfortunately still a circular import that way, or I'd also have had it at the top 👌